fl4health/clients/basic_client.py

```diff
@@ -699,6 +700,8 @@ def validate(self) -> Tuple[float, Dict[str, Scalar]]:
                 self.test_loader, self.test_loss_meter, self.test_metric_manager, LoggingMode.TEST
             )
             # There will be no clashes due to the naming convention associated with the metric managers
+            if self.num_test_samples is not None:
+                val_metrics["test - num_examples"] = int(self.num_test_samples)
```

Comment:
Perhaps we should formalize the "test -" prefix as an enum in utils and import it here and in the base server just to ensure that they don't get out of sync at some point later on?

Reply:
test - here is more folloiwing the convention of self.test_metric_manager = MetricManager(metrics=self.metrics, metric_manager_name="test"), maybe rather than adding an enum would it make for sense to use metric_manager_name instead?